forms-example/src/main/java/com/stackhunter/blog/example/articlesubmission/website/ArticlesResource.java

```diff
@@ -24,7 +24,7 @@
 import java.util.UUID;
 import java.util.concurrent.ConcurrentHashMap;
 
-import javax.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletRequest;
 import javax.ws.rs.FormParam;
 import javax.ws.rs.GET;
 import javax.ws.rs.POST;
```

Comment:
Do these need to be converted to `Jakarta` as well?

Reply:
Updated it. Done